newa/__init__.py

```diff
@@ -1,14 +1,19 @@
+from __future__ import annotations
+
 import io
 from enum import Enum
 from pathlib import Path
 from typing import TYPE_CHECKING, Any, Optional, TypeVar
 
 import attrs
 import jinja2
+import requests
 import ruamel.yaml
 import ruamel.yaml.nodes
 import ruamel.yaml.representer
+import time
 from attrs import define, field
+from requests_kerberos import HTTPKerberosAuth
```

Comment:
Both `requests` and `requests_kerberos` should be added to `pyproject.toml`, https://github.com/RedHatQE/newa/blob/main/pyproject.toml#L39

Reply:
Ah, sorry - fixed now.